youtube_dl/extractor/wdr.py

```diff
@@ -239,6 +241,59 @@ def _real_extract(self, url):
         return info_dict
 
 
+class WDRElefantIE(WDRBaseIE):
+    _VALID_URL = r'https?://(?:www\.)wdrmaus.de/elefantenseite/#(?P<display_id>.+)'
+    IE_NAME = 'wdr:elefant'
+
+    _TESTS = [
+        {
+            'url': 'http://www.wdrmaus.de/elefantenseite/#lieder_geburtstagslied',
+            'info_dict': {
+                'title': 'Ich bin schon 1-2-3',
+                'id': 'mdb-1008774',
+                'ext': 'mp4',
+                'age_limit': None,
+                'upload_date': '20091119'
+            },
+        },
+        {
+            'url': 'http://www.wdrmaus.de/elefantenseite/#folge_ostern_2015',
+            'info_dict': {
+                'title': 'Folge Oster-Spezial 2015',
+                'id': 'mdb-1088195',
+                'ext': 'mp4',
+                'age_limit': None,
+                'upload_date': '20150406'
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        display_id = mobj.group('display_id')
```

Comment:
`self._match_id`.

Reply:
Sorry, I don't understand. Do you mean I should use "self._match_id" instead of "mobj.group" in the last line? How would that work, given that "self._match_id" is for matching the id, and not the display_id?

Reply:
Nothing stops from renaming capture group to `id`.

Reply:
Yes, of course I could do that. But as far as I see, the string I'm retrieving here is not the id. The id is retrieved later, in line 95 ( metadata_tracker_data.get('trackerClipId', display_id) ). So by calling this string "id" instead of "display_id" I would just cause confusion, wouldn't I?
If you insist, I'll gladly change "display_id" to "id" (everywhere in the code, otherwise it would be even more confusing) - I'm just not sure that would be helpful.